common/cards/default/hermits/geminitay-rare.ts

```diff
@@ -31,26 +31,29 @@ class GeminiTayRareHermitCard extends HermitCard {
 	override onAttach(game: GameModel, instance: string, pos: CardPosModel) {
 		const {player} = pos
 
-		player.hooks.afterAttack.add(instance, (attack) => {
+		player.hooks.onAttack.add(instance, (attack) => {
 			if (attack.id !== this.getInstanceKey(instance) || attack.type !== 'secondary') return
 
-			// To keep this simple gem will discard the single use card, if it's used
-			if (player.board.singleUseCardUsed) {
+			player.hooks.afterAttack.add(instance, (attack) => {
+				// To keep this simple gem will discard the single use card, if it's used
```

Comment:
As this is no longer accurate, just remove the "if it's used", then it's good to go.